mypy/subtypes.py

```diff
@@ -1199,6 +1213,18 @@ def report(*args: Any) -> None:
     return applied
 
 
+def try_restrict_literal_union(t: UnionType, s: Type) -> Optional[List[Type]]:
+    """Helper function for restrict_subtype_away, allowing a fast path for Union of simple literals"""
```

Comment:
This docstring doesn't say what the function does, which is the part I was having trouble with. What are the parameters and what does it return?

Reply:
Done